src/tooltip/tooltip.spec.ts

```diff
@@ -32,6 +32,15 @@ describe('ngb-tooltip-window', () => {
     fixture.detectChanges();
     expect(fixture.nativeElement).toHaveCssClass('tooltip-left');
   });
+
+  it('should have custom classes assigned', () => {
+    const fixture = TestBed.createComponent(NgbTooltipWindow);
+    fixture.componentInstance.customClasses = 'custom-class1 custom-class2';
+    fixture.detectChanges();
+    expect(fixture.nativeElement).toHaveCssClass('tooltip-top');  // default class
```

Comment:
I don't know that this is strictly necessary since there are three other tests that check for this class.  I'm ok with removing it.